src/runner.py

```diff
@@ -132,6 +133,8 @@ def loop(self):
                 self._cleanup_paths()
         except KeyboardInterrupt as e:
             self._logger.log_message(logging.INFO, "Stopping.")
+        except Exception as e:
```

Comment:
The same comment as above for other services.

Reply:
The same in line 133/134